docs/Using-EthSigner/Hashicorp.md

```diff
@@ -0,0 +1,56 @@
+description: Signing transactions with key stored in Hashicorp Key Vault
+<!--- END of page meta data -->
+
+# Using EthSigner with Hashicorp Vault 
+
+EthSigner supports storing the signing key in a [Hashicorp Vault](https://www.hashicorp.com/products/vault/). 
+
+!!! caution 
+    EthSigner with Hashicorp Vault does not support TLS. If using a remote Hashicorp Vault, apply appropriate 
+    security. 
+
+## Storing Private Key in Hashicorp Vault 
+
+After installing [Hashicorp Vault](https://learn.hashicorp.com/vault/getting-started/install) and
+[starting the server](https://learn.hashicorp.com/vault/getting-started/dev-server):
+
+1.  Set the `VAULT_ADDR` environment variable using the command displayed after starting the server: 
+    ```bash
+    export VAULT_ADDR='http://127.0.0.1:8200'
+    ```
+
+2. Save the root token displayed after started the server in a file called `authFile`. 
+
+3. Put your signing key into the Hashicorp Vault: 
+
+    ```bash tab="Command"
+    vault kv put secret/ethsignerSigningKey value=<Private Key ex 0x)
+    ```
+       
+    ```bash tab="Example"
+    vault kv put secret/ethsignerSigningKey value=8f2a55949038a9610f50fb23b5883af3b4ecb3c3bb792cbcefbd1542c692be63
+    ```
+
+    The private key is stored in the default location for EthSigner. 
+
+## Start Pantheon 
+
+[Start Pantheon](https://docs.pantheon.pegasys.tech/en/stable/Getting-Started/Starting-Pantheon/) with the 
+[`--rpc-http-port`](https://docs.pantheon.pegasys.tech/en/stable/Reference/Pantheon-CLI-Syntax/#rpc-http-port)
+option set to `8590`. 
```

Comment:
```suggestion
option set to `8590` to prevent conflict with EthSigner that will run on `8545` by default. 
```

Reply:
As the EthSigner port is a configuration option ...EthSigner _could_ be running on `8590` 😁

Reply:
I do approve of adding a reason / rational for choosing `8590` (i.e. used / non-conflicting port).

Reply:
Updated. Added a tip to make it clear the EthSigner port can also be changed.